packages/playground/src/weblets/profile_manager.vue

```diff
@@ -271,7 +271,7 @@
             color="primary"
             variant="text"
             @click="updateSSH"
-            :disabled="!ssh || profileManager.profile.ssh === ssh || updatingSSH"
+            :disabled="!ssh || profileManager.profile.ssh === ssh || updatingSSH || !isEnoughBalance(balance)"
```

Comment:
```suggestion
            :disabled="!ssh || profileManager.profile.ssh === ssh || updatingSSH || !isEnoughBalance(balance, 0.001)"
```

Please update isEnoughBalance to  take the balance and the amount to check for because right now no idea what is the enough balance without digging into isEnoughBalance ...

Reply:
Wouldn't that leave room for error if the second param does not equal the lowest possible balance? The minimum balance should be the same in all cases and the suggested fix won't ensure that.

Reply:
Added a minimum amount by default in isEnoughBalance.